ports/esp32s2/boards/adafruit_funhouse_esp32s2/board.h

```diff
@@ -69,37 +69,36 @@
 #define DISPLAY_PIN_MOSI      35
 #define DISPLAY_PIN_SCK       36
 
-#define DISPLAY_PIN_CS        39
-#define DISPLAY_PIN_DC        38
-#define DISPLAY_PIN_RST       40
+#define DISPLAY_PIN_CS        40
+#define DISPLAY_PIN_DC        39
+#define DISPLAY_PIN_RST       41
 
 #define DISPLAY_PIN_BL        21
 #define DISPLAY_BL_ON         1  // GPIO state to enable back light
 
 #define DISPLAY_WIDTH         240
 #define DISPLAY_HEIGHT        240
 
-#define DISPLAY_COL_OFFSET    0 // 53
-#define DISPLAY_ROW_OFFSET    0 // 40
+#define DISPLAY_COL_OFFSET    0
+#define DISPLAY_ROW_OFFSET    0
 
 // Memory Data Access Control & // Vertical Scroll Start Address
 #define DISPLAY_MADCTL        (TFT_MADCTL_MX | TFT_MADCTL_MY | TFT_MADCTL_MV)
 #define DISPLAY_VSCSAD        80
 
 #define DISPLAY_TITLE         "Fun House"
 
-
 //--------------------------------------------------------------------+
 // USB UF2
 //--------------------------------------------------------------------+
 
 #define USB_VID           0x239A
-#define USB_PID           0x00E5
+#define USB_PID           0x00E5 // TODO need PID for its own
```

Comment:
need new PID, this one is shared with magtag 2.9

Reply:
Adafruit FunHouse | bootloader | 0x239A | 0x00F9